monai/bundle/scripts.py

```diff
@@ -293,7 +300,7 @@ def load(
     configer = ConfigComponent(config=net_kwargs)
     model = configer.instantiate()
     model.to(device)  # type: ignore
-    model.load_state_dict(model_dict)  # type: ignore
+    copy_model_state(dst=model, src=model_dict if key_in_ckpt is None else model_dict[key_in_ckpt])  # type: ignore
```

Comment:
do you think this util should be used somewhere in this function? https://github.com/Project-MONAI/MONAI/blob/2bd56dc7b35422a60fbd39bed38e094c423a986b/monai/networks/utils.py#L383-L394

Reply:
Hi @wyli ,

I think this PR refers to: https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/scripts.py#L592.
The `copy_model_state()` API leverages the `get_state_dict` as you mentioned.

Thanks.

Reply:
sure, in that case calling `model_dict[key_in_ckpt]` here might be unstable, because it's assuming a dictionary?

Reply:
Thanks @wyli . As defined in https://docs.monai.io/en/latest/mb_specification.html `model.pt` is " the state dictionary of a saved model". Could we just add some more descriptions about it?

Reply:
I think this is confusing, I'd suggest:

```py
model_dict = torch.load...
if not isinstance(model_dict, Mapping):
    warnings.warn(f"the state dictionary from {full_path} should be a dictionary but go {type(model_dict}.")
    model_dict = get_state_dict(model_dict)

```